distributed/nanny.py

```diff
@@ -162,7 +162,8 @@ def __init__(
         self.death_timeout = parse_timedelta(death_timeout)
 
         self.Worker = Worker if worker_class is None else worker_class
-        self.env = env or {}
+        self.env = {**dask.config.get("distributed.nanny.environ"), **env}
```

Comment:
Should we also pull in the local environment to ensure that takes precident?

```suggestion
        self.env = {**dask.config.get("distributed.nanny.environ"), **env, **os.environ}
```

Reply:
I think that precedence here becomes interesting.  I would certainly want `**env` to take precedence.  Even between config and os.environ it's an open question.  If someone sets an environment variable on the command line then sure, but what about if it's set in bashrc?  I might think differently then.

Reply:
However, I've made an edit with something similar to your suggestion.  I'm ambivalent on this point.